### PR TITLE
Added german translation

### PR DESCRIPTION
**Pull request by olagino**
I did my best to help translating the messages into german, I hope, my pull request is conform to the contributing guidelines.

### CODE REVIEW

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -3,340 +3,350 @@
...
 
 #: build/models.py:78
 msgid "Build status"
-msgstr ""
+msgstr "Bau-Status"
 
 #: build/models.py:81
 msgid "Batch code for this build output"
-msgstr ""
+msgstr "Stapel-Code für diese Bau-Ausgabe"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: Perhaps "Losbezeichnung" or "Chargennummer"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3,340 +3,350 @@
...
 
 #: build/models.py:93
 msgid "Link to external URL"
-msgstr ""
+msgstr "Link zu einer Externen URL"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "externen"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -3,340 +3,350 @@
...
 #: part/templates/part/detail.html:33
 msgid "Description"
-msgstr ""
+msgstr "Beschreibung"
 
 #: build/templates/build/allocate_view.html:22
 #: part/templates/part/part_base.html:122
 msgid "On Order"
-msgstr ""
+msgstr "auf Bestellung"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "bestellt" or "rückständig" or something like that?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -3,340 +3,350 @@
...
 #, python-brace-format
 msgid "The following serial numbers already exist: ({sn})"
-msgstr ""
+msgstr "Die folgende Seriennummer existiert bereits: ({sn})"
 
 #: common/models.py:65
 msgid "Settings key (must be unique - case insensitive"
 msgstr ""
+"Einstellungs-Schlüssel (muss einzigartig sein, Groß-/ Kleinschreibung "
+"beachten)"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: case **in**sensitive

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -3,340 +3,350 @@
...
 
 #: order/forms.py:32
 msgid "Mark order as complete"
-msgstr ""
+msgstr "Bestellung als vollständig markieren"
 
 #: order/forms.py:43
 msgid "Cancel order"
-msgstr ""
+msgstr "Bestellung abbrechen"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "stornieren"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 #: part/forms.py:73
 msgid "Select part category"
-msgstr ""
+msgstr "Teilekategorie wählen"
 
 #: part/forms.py:81
 msgid "Perform 'deep copy' which will duplicate all BOM data for this part"
 msgstr ""
+"\"Deep Copy\" ausführen. Dies wird alle Daten der Stückliste für dieses "
+"Teil duplizieren"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "tiefe Kopie" for deep copy is an established term, I think.

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/models.py:423
 msgid "Is this a virtual part, such as a software product or license?"
-msgstr ""
+msgstr "Ist dieses Teil virtuell, wie zum Beispiel eine Software oder Lizenz?"
 
 #: part/models.py:427
 msgid "Stored BOM checksum"
-msgstr ""
+msgstr "Kontrollsumme der Stückliste gespeichert"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Prüfsumme"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/models.py:998
 msgid "Parameter Units"
-msgstr ""
+msgstr "Parameter Einheit"
 
 #: part/models.py:1024
 msgid "Parent Part"
-msgstr ""
+msgstr "Augangsteil"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: Missing s

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/models.py:1066
 msgid "BOM quantity for this BOM item"
-msgstr ""
+msgstr "Stücklisten-Anzahl für dieses Stücklisten-Teil"
 
 #: part/models.py:1069
 msgid "Estimated build wastage quantity (absolute or percentage)"
-msgstr ""
+msgstr "Geschätzte Bau-Fehleranzahl (absolut oder prozentual)"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Geschätzter Ausschuss (absolut oder prozentual)"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/models.py:1072
 msgid "BOM item reference"
-msgstr ""
+msgstr "Position des Objekts auf der Stückliste"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Referenz des..."

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/models.py:1075
 msgid "BOM item notes"
-msgstr ""
+msgstr "Notizen zum Stücklisten-Objekt"
 
 #: part/models.py:1077
 msgid "BOM line checksum"
-msgstr ""
+msgstr "Kontrollsumme der Stückliste"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Prüfsumme"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 #: part/models.py:1140
 msgid "Part cannot be added to its own Bill of Materials"
-msgstr ""
+msgstr "Teil kann nicht zu seiner eigenen Stückliste hinzugefügt werden"
 
 #: part/models.py:1147
 #, python-brace-format
 msgid "Part '{p1}' is  used in BOM for '{p2}' (recursive)"
-msgstr ""
+msgstr "Teil '{p1}' wird in Stückliste für Teil '{p2}' benutzt (Rekursiv)"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "rekursiv" lower case

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/templates/part/category.html:13 part/templates/part/category.html:69
 msgid "Part Categories"
-msgstr ""
+msgstr "Teile-Kategorie"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: Missing n

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/templates/part/category.html:41
 msgid "Category Description"
-msgstr ""
+msgstr "Kategorie-Beschreibung"
 
 #: part/templates/part/category.html:46 part/templates/part/detail.html:64
 msgid "Default Location"
-msgstr ""
+msgstr "Standard-Standort"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Standard-Lagerort"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/templates/part/category.html:73
 msgid "Parts"
-msgstr ""
+msgstr "Teile"
 
 #: part/templates/part/detail.html:22 part/templates/part/part_base.html:85
 msgid "IPN"
-msgstr ""
+msgstr "IPN (Interne Produkt Nummer)"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Produktnummer"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 16]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/templates/part/detail.html:91
 msgid "Virtual"
-msgstr ""
+msgstr "Virtuell"
 
 #: part/templates/part/detail.html:94
 msgid "Part is virtual (not a physical part)"
-msgstr ""
+msgstr "Teil ist Virtuell (kein physisches Teil)"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "virtuell" lower case

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 17]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: part/templates/part/detail.html:114
 msgid "Part cannot be used in assemblies"
-msgstr ""
+msgstr "Teil kann nicht in Baugruppen benutzt werden"
 
 #: part/templates/part/detail.html:118
 msgid "Trackable"
-msgstr ""
+msgstr "auffindbar"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Nachverfolgbar"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 18]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: stock/models.py:320
 msgid "Serial number for this item"
-msgstr ""
+msgstr "Seriennummer für dieses Teil"
 
 #: stock/models.py:325
 msgid "Batch code for this stock item"
-msgstr ""
+msgstr "Stapel-Code für dieses Lagerobjekt"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Losnummer"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 19]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 #: stock/models.py:334
+#, fuzzy
 msgid "Build for this stock item"
-msgstr ""
+msgstr "Bau für dieses Lagerobjekt"
 
 #: stock/models.py:343
 msgid "Purchase order for this stock item"
-msgstr ""
+msgstr "Bestellung für dieses Teil absenden"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "~absenden~"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 20]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: stock/models.py:361
 msgid "Stock Item Notes"
-msgstr ""
+msgstr "Lagerobjekt-Notizen"
 
 #: stock/models.py:435
 msgid "Quantity must be integer"
-msgstr ""
+msgstr "Anzahl muss eine Ganz-Zahl sein"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Ganzzahl"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 21]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 #: stock/models.py:441
 #, python-brace-format
 msgid "Quantity must not exceed available stock quantity ({n})"
-msgstr ""
+msgstr "Anzahl darf nicht die verfügbare Anzahl überschreiten ({n})"
 
 #: stock/models.py:444 stock/models.py:447
 msgid "Serial numbers must be a list of integers"
-msgstr ""
+msgstr "Seriennummern muss eine Liste von Ganz-Zahlen sein"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Ganzzahlen"?

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 22]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: stock/models.py:750
 msgid "Tracking entry title"
-msgstr ""
+msgstr "Name des Eintrags-Trackings"
 
 #: stock/models.py:752
 msgid "Entry notes"
-msgstr ""
+msgstr "Eingrags-Notizen"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Eintrags-Notizen"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 23]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: stock/templates/stock/item.html:84
 msgid "Serial Number"
-msgstr ""
+msgstr "Seriennummer"
 
 #: stock/templates/stock/item.html:95
 msgid "Batch"
-msgstr ""
+msgstr "Stapel"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "Los"

File: InvenTree/locale/de/LC_MESSAGES/django.po [hunk 24]
```diff
@@ -348,926 +358,940 @@ msgstr ""
...
 
 #: stock/views.py:379
 msgid "Delete Stock Items"
-msgstr ""
+msgstr "Lagerobjekte löschen"
 
 #: stock/views.py:407
 msgid "Must enter integer value"
-msgstr ""
+msgstr "Nur Ganzzahl Objekte eingeben"
```
Comment (chschlue) [inline InvenTree/locale/de/LC_MESSAGES/django.po]: "~Objekte~"

**Comment by olagino** [comment]
Thank you for your review, @chschlue! I missed quite a few typos :cry: and also thank you for the alternative suggestions too!

**Comment by chschlue** [comment]
@SchrodingersGat looks good to me now.

**Comment by SchrodingersGat** [comment]
Thanks for the contribution @olagino and the review @chshlue :) Fantastic work

**Pull request merged**